test/test_model.py

```diff
@@ -70,10 +82,27 @@ def test_relations_keys(self):
     def test_get_relation(self):
         err_msg = 'ERROR invalid value "$2" for option -r: relation not found'
 
-        fake_script(self, 'relation-ids',
-                    """([ "$1" = db1 ] && echo '["db1:4"]') || ([ "$1" = db2 ] && echo '["db2:5", "db2:6"]') || echo '[]'""")
-        fake_script(self, 'relation-list',
-                    """([ "$2" = 4 ] && echo '["remoteapp1/0"]') || (echo {} >&2 ; exit 2)""".format(err_msg))
+        fake_script(self, 'relation-ids', '''
```

Comment:
I'm working through a patch that changes these to TestingHarness, mostly looks nicer.
But before I finish, you can land yours... :)